lib/dialects/oracle/transaction.js

```diff
@@ -40,38 +40,4 @@ module.exports = class Oracle_Transaction extends Transaction {
       });
   }
 
-  acquireConnection(config, cb) {
-    const configConnection = config && config.connection;
-    return new Bluebird((resolve, reject) => {
-      try {
-        resolve(configConnection || this.client.acquireConnection());
-      } catch (e) {
-        reject(e);
-      }
-    })
-      .then((connection) => {
-        connection.__knexTxId = this.txid;
-
-        return connection;
-      })
-      .then((connection) => {
-        if (!this.outerTx) {
-          connection.setAutoCommit(false);
-        }
-        return connection;
-      })
-      .then(async (connection) => {
-        try {
-          return await cb(connection);
-        } finally {
-          debugTx('%s: releasing connection', this.txid);
-          connection.setAutoCommit(true);
-          if (!configConnection) {
-            this.client.releaseConnection(connection);
-          } else {
-            debugTx('%s: not releasing external connection', this.txid);
-          }
-        }
-      });
-  }
```

Comment:
Dead Code.

Reply:
As in: this was already dead code before this refactoring started.

Reply:
Why this is dead code?

Reply:
There are a few reasons:

1. `"oracle"` is not listed within the `SUPPORTED_CLIENTS` constant:

https://github.com/knex/knex/blob/c9be84890cf7d3211c6103d7c914a60b4b84647b/lib/constants.js#L10-L20

2. The `lib/dialects/oracle` module is only referenced by the `lib/dialects/oracledb` module.
3. The `lib/dialects/oracledb` module sub-classes this code and then overrides the `acquireConnection(..)` method.

Additionally, tests continue to succeed even after this code has been removed.